microsoft/onefuzz
# Add dotnet coverage task

## Summary of the Pull Request

_What is this about?_

## PR Checklist

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

1. Build a dotnet project
2. Include dll, pdb, and runtimeconfig.json in setup container
3. Add some files in the inputs container
4. Create a new job
5. Create a new task. Example config: `onefuzz tasks create 4e43e15a-b003-4d5d-bc37-7907f0a7d527 dotnet_coverage testcoverage.dll coverage=coverage setup=setup readonly_inputs=inputs linux --target_options {input}`
6. Verify a cobertura-coverage.xml file is in the coverage container
7. Upload another file in the input container
8. Verify the cobertura-coverage.xml file was updated
